test/integration/docs.test.js

```diff
@@ -17,7 +20,17 @@ describe('api-docs', function () {
 
     expect(actualResult.status).to.equal(200)
     expect(actualResult.body).to.be.a.jsonObj()
-    expect(JSON.stringify(actualResult.body)).to.include('openapi')
-    expect(JSON.stringify(actualResult.body)).to.include('info')
+
+    var validator = new OpenAPISchemaValidator({
+      version: 3,
+      // optional
+      extensions: {
+        /* place any properties here to extend the schema. */
+      },
+    })
```

Comment:
I'd build the validator outside the describe. That way the test assertion will be clearer